Manifest/styles/_header.scss

```diff
@@ -0,0 +1,74 @@
+.header {
+    width: 100%;
+    height: 60px;
+    background-color: $white;
+    @include flex(row);
+    justify-content: space-between;
+    align-items: center;
+    padding-left: 10px;
+    padding-right: 10px;
+    @media screen and (min-width: 768px) {
+        padding-left: 30px;
+        padding-right: 30px;
+    }
+    @media screen and (min-width: 1440px) {
+        height: 100px;
+    }
+    .logo {
+        width: 145px;
+    }
+    .main-menu {
+        width: 90%;
+        @include flex(row);
+        justify-content: flex-end;
+        ul {
+            width: 60%;
+            @include flex(row);
+            justify-content: space-between;
+            list-style-type: none;
+
+            overflow: hidden;
+            li a {
+                display: inline-block;
+                color: $black;
+                font-family: $lato;
+                text-transform: uppercase;
+                text-decoration: none;
+            }
+            li a:hover {
+                opacity: 0.6;
+            }
+        }
+        .default {
+            text-decoration: underline;
+        }
+        @media screen and (max-width: 1440px) {
+            display: none;
+        }
+    }
+    .right-menu {
+        @include flex(row);
+        justify-content: flex-end;
+        @media screen and (min-width: 768px) {
+            width: 33%;
+        }
+        .search {
```

Comment:
why class is inside ??
should be inside if it would be named .right-menu__search and use as __search
search is independent class so should not be inside 

refer to all other classses